scripts/go/configs/golangci.yml

```diff
@@ -37,19 +37,59 @@ linters:
     - varcheck
     - whitespace
 
-linter-settings:
+linters-settings:
   goconst:
-    ignore-tests: true
+    ignore-tests: false
     min-len: 5
     min-occurrences: 5
   gocyclo:
-    min-complexity: 10
+    min-complexity: 20
```

Comment:
we can progressively lower this down to 10

Reply:
can we add a TODO for this please :)